apolloschurchapp/appcenter-pre-build.sh

```diff
@@ -6,7 +6,15 @@ cp $droid_dir/gradle.properties.appcenter $droid_dir/gradle.properties
 
 cd ios
 pod install
+cd ..
 
-cd ../../..
-yarn
-yarn lerna run generate-stories
+# You can comment this out if your app doesn't have an existing build number.
+VERSION_CODE=$((VERSION_CODE_SHIFT + APPCENTER_BUILD_ID))
+
+echo Using "$VERSION_CODE" as build number
+
+echo $VERSION_CODE
```

Comment:
This is something that all our partners have needed a solution for. This lets us use both incremental build numbers in app center, and avoid running into the "build number must be greater than last build number" issue. (AppCenter always starts at 1 for incrementing build numbers)